routes/groups.js

```diff
@@ -1,5 +1,6 @@
 const router = require('express').Router({ mergeParams: true });
 const groups = require('../controllers/groups_controller');
+const groupmemberships_routes = require('./groupmemberships');
```

Comment:
```suggestion
const group_memberships_routes = require('./group_memberships');
```

Can we standardise to snake case?